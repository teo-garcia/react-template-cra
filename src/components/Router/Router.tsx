import { Route, Routes } from 'react-router-dom';
import { Home } from '../../pages';

function Router() {
  return (
    <Routes>
      <Route path="/" element={<Home />} />
    </Routes>
  );
}

export default Router;
