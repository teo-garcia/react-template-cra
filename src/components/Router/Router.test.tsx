import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Router from './Router';

describe('<Router /> tests', function () {
  test('Should match with the component snapshot', function () {
    const { asFragment } = render(
      <BrowserRouter>
        <Router />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
