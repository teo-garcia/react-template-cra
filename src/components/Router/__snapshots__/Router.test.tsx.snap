// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Router /> tests Should match with the component snapshot 1`] = `
<DocumentFragment>
  <section
    class="message"
  >
    <h1
      class="typography typography--bold typography--text message__brand"
    >
      PING
    </h1>
    <p
      class="typography typography--regular typography--neutral message__content"
    >
      We are launching soon!
    </p>
    <p
      class="typography typography--regular typography--text message__subcontent"
    >
      Subscribe and get notified
    </p>
  </section>
  <form
    class="subscribe-form"
  >
    <label
      class="subscribe-form__label"
      for="email"
    >
      Your email address:
    </label>
    <input
      aria-describedby="error-email"
      class="subscribe-form__input"
      id="email"
      name="email"
      value=""
    />
    <button
      class="button subscribe-form__cta"
    >
      Notify Me
    </button>
  </form>
  <img
    alt="Ping dashboard preview"
    src="dashboard.png"
  />
</DocumentFragment>
`;
